modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -66,9 +76,13 @@ def get(self):
         pandas.DataFrame
             The object from the Plasma store.
         """
+        logger = get_logger()
```

Comment:
Shouldn't we be using the logger_function + logger decorator for these things?

Reply:
The function and decorator really only logs the timestamp for when the top-level `get` function starts and ends, but we want to log at a finer level of granularity here inside of the function.